test/user.js

```diff
@@ -255,14 +255,3 @@ describe('reset password with an unexisting email', () => {
       .catch(error => logError(error));
   });
 });
-
```

Comment:
Hi frank the tests for resetting the password is not mine, it's yours maybe you have forgotten that you suggested that the test to reset password be put in article file in order to use the user I created in article tests because it wasn't stable and was failing repeatedly thx.